examples/vl3_basic/helm/vl3/templates/vl3-nse-ucnf.tpl

```diff
@@ -25,7 +25,11 @@ spec:
     spec:
       containers:
         - name: vl3-nse
-          image: {{ .Values.registry }}/{{ .Values.org }}/vl3_ucnf-vl3-nse:{{ .Values.tag }}
+{{- if .Values.image }}
+          image: {{ .Values.registry }}/{{ .Values.org }}/{{ .Values.image }}:{{ .Values.tag }}
+{{- else }}
+          image: {{ .Values.registry }}/{{ .Values.org }}/vl3_ucnf-{{ .Values.nsm.serviceName }}:{{ .Values.tag }}
```

Comment:
You don't want specific image names for different services.  Please change the image name back.